Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:5-slim
+
+RUN apt-get update
+RUN apt-get install libpng12-0
```

Comment:
`update` and `install` should be in the same `RUN`, and clean up apt afterwards.

Like this:
```
RUN apt-get update && apt-get install -y --no-install-recommends \
		libpng12-0
	&& rm -rf /var/lib/apt/lists/*
```

See https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/apt-get